ee/debug/checkups/checkpoint.go

```diff
@@ -72,5 +77,25 @@ func (c *logCheckPointer) Once(ctx context.Context) {
 			"data", checkup.Data(),
 			"status", checkup.Status(),
 		)
+
+		switch checkup.Status() {
+		case Warning:
+			warningCount += 1
+		case Failing:
+			failingCount += 1
+		case Passing:
+			passingCount += 1
+		case Erroring, Informational, Unknown:
+			// Nothing to do here
```

Comment:
I think we should track Erroring and Unknown. Possible as two more metrics. They represent problem with our system that we _ought_ address. (But not a device health score)

Reply:
Looks like we only use `Unknown` right now when the launcher installation is not Kolide-hosted -- maybe not worth tracking yet? I'll add a gauge for `Erroring`!

Reply:
Oh, ha, so currently none of our `logSupported` checkups set `Erroring` status. Added anyway, we'll use it at some point!